### PR TITLE
Refine sections in tikv-configuration-file.md

### PR DESCRIPTION
**Pull request by pepezzzz**
according to tikv config-template.toml, merge grpc section to new server section

### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

Fix the mismatch grpc section. Add the [server] section. 
Close https://github.com/pingcap/docs/issues/4510.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v5.0 (TiDB 5.0 versions)
- [x] v4.0 (TiDB 4.0 versions)
- [x] v3.1 (TiDB 3.1 versions)
- [x] v3.0 (TiDB 3.0 versions)
- [x] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by TomShawn** [comment]
@pepezzzz Would you please involve a technical review?

**Comment by andylokandy** [comment]
Could you fix this in the pr:

https://github.com/pingcap/docs/issues/4510

File: tikv-configuration-file.md [hunk 1]
```diff
@@ -78,31 +42,68 @@ TiKV 配置文件比命令行参数支持更多的选项。你可以在 [etc/con
...
+### `end-point-request-max-handle-duration`
+
++ endpoint 下推查询请求处理任务最长允许的时长。
++ 默认值：60s
++ 最小值：1s
+
+### `snap-max-write-bytes-per-sec`
+
++ 处理 snapshot 时最大允许使用的磁盘带宽。
++ 默认值：1000MB
```
Comment (TomShawn) [inline tikv-configuration-file.md:97]:
```suggestion
+ 默认值：100MB
```

**Comment by TomShawn** [comment]
> Could you fix this in the pr:
> 
> [pingcap/docs#4510](https://github.com/pingcap/docs/issues/4510)

Done.

**Review by andylokandy — APPROVED** [review]

File: tikv-configuration-file.md [hunk 2]
```diff
@@ -10,51 +10,15 @@ TiKV 配置文件比命令行参数支持更多的选项。你可以在 [etc/con
 本文档只阐述未包含在命令行参数中的参数，命令行参数参见 [TiKV 配置参数](/command-line-flags-for-tikv-configuration.md)。
 
 <!-- markdownlint-disable MD001 -->
+
+## server
+
```
Comment (TomShawn) [inline tikv-configuration-file.md:15]:
```suggestion

服务器相关的配置项。

```

**Pull request merged**